sources/reference/api/experiment.md

```diff
@@ -255,6 +263,48 @@ A jsonpath tolerance with an expected value to match:
 }
 ```
 
+### Contributions
+
+Contributions describe the facets targeted by this particular
+experiment, as well as a weight for each facet. Those facets usually refer to
```

Comment:
As well as a weight for each contribution, as in "how much this experiment contributes to trust in confidence in this valuable system property"